src/PerformanceTests/Common/Scenarios/BaseRunner.cs

```diff
@@ -100,53 +90,40 @@ private void CreateSeedData()
                 if (i % 5000 == 0)
                     Log.InfoFormat("Seeded {0} messages.", i);
 
-                ((ICreateSeedData)this).SendMessage(sendonlyInstance);
+                ((ICreateSeedData)this).SendMessage(Session);
```

Comment:
SendMessage returns a Task. You have to await it or use GetAwaiter().GetResult()